feat(hook): Add id product attribute to hook

Kaikina opened the pull request:
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | It makes sense to add the id product attribute into the hook params.
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | NA
| How to test?  | Register this hook, access the passed array and test that id product attribute key is set.

Kaikina committed:
feat(hook): Add instance of MailAlert to hook

ps-jarvis commented:
<!-- PR_WELCOME -->
Hello @Kaikina!

This is your first pull request on [ps_emailalerts](https://github.com/PrestaShop/ps_emailalerts/) repository of the PrestaShop project.

Thank you, and welcome to this Open Source community!

boherm reviewed (dismissed):
Thanks for your contribution! I agree with you, that could be nice to have :)

Kaikina committed:
feat(hook): Add id_product_attribute to hook

Kaikina commented:
I updated to "id_product_attribute". I didn't see it was a static function (I don't like static functions). So no way of passing "$this". But actually, I think the only missing information while using the hook was the id_product_attribute, so I added it.

Kaikina commented:
May we merge this branch ?

kpodemski reviewed (dismissed):
Wrong indentation.

jf-viguier reviewed (changes requested):
@Kaikina Please update PR description, it's not MailAlert instance anymore.

Kaikina committed:
Update MailAlert.php

Kaikina commented:
@kpodemski @jf-viguier updated.

jf-viguier committed:
Fix indentation for id_product_attribute assignment
